modules/consensus/synchronize_test.go

```diff
@@ -438,12 +438,12 @@ func TestIntegrationRPCSendBlocks(t *testing.T) {
 		// Create the "remote" peer.
 		remoteCST, err := blankConsensusSetTester(filepath.Join(t.Name()+" - remote", strconv.Itoa(i)))
 		if err != nil {
-			t.Fatal(err)
+			t.Fatalf("%v: %s", tt.msg, err)
```

Comment:
%s works. From https://golang.org/pkg/fmt/

>  If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).

But if %v is used in other parts of the code, it makes sense to use it here as well. I'll amend my commit.